HoneyDo.Web/ClientApp/src/registerServiceWorker.js

```diff
@@ -93,6 +93,7 @@ function checkValidServiceWorker(swUrl) {
       }
     })
     .catch(() => {
+      // eslint-disable-next-line no-console

```

Comment:
Albeit not recommended to commit these, it's handy to have for reasons since we don't have proper logging setup. (They will be removed in my next PR that adds some components and moves to Material UI).